src/dom.js

```diff
@@ -435,16 +375,54 @@ export function childElementByTag(parent, tagName) {
  * @return {!Array<!Element>}
  */
 export function childElementsByTag(parent, tagName) {
+  return scopedQuerySelectorAll(parent, `> ${tagName}`);
+}
+
+
+/**
+ * Finds the first element that matches `selector`, scoped inside `root`.
+ * Note: in IE, this causes a quick mutation of the element's class list.
+ * @param {!Element} root
+ * @param {string} selector
+ * @return {?Element}
+ */
+export function scopedQuerySelector(root, selector) {
```

Comment:
We initially decided to hold this off because of https://bugs.chromium.org/p/chromium/issues/detail?id=222028